src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleGroupSyncMonitor.scala

```diff
@@ -28,15 +28,15 @@ object GoogleGroupSyncMonitorSupervisor {
   case object Start extends GoogleGroupSyncMonitorSupervisorMessage
 
   def props(
-      pollInterval: FiniteDuration,
-      pollIntervalJitter: FiniteDuration,
-      pubSubDao: GooglePubSubDAO,
-      pubSubTopicName: String,
-      pubSubSubscriptionName: String,
-      workerCount: Int,
-      groupSynchronizer: GoogleGroupSynchronizer): Props =
+             pollInterval: FiniteDuration,
+             pollIntervalJitter: FiniteDuration,
+             groupSyncPubSubDao: GooglePubSubDAO,
```

Comment:
I renamed the Daos in the monitors to make it clear which pubSubDAO they are using.